minio/test_file_head.sh

```diff
@@ -0,0 +1,10 @@
+bucket_name="limaos90"
+resource_id="977c24b20a1a4c3caf71bf7e47ff802a"
+file_name="maps/etp/etp00000.219"
+
+
+if ! file_head=$(mc head -n 1 prod-minio/$bucket_name/$resource_id/data/contents/$file_name 2>/dev/null); then
```

Comment:
`mc head -n 1` for the win!
NICE -- I wouldn't have thought of that and we would be iterating files for years...